packages/ui/toolbar/src/ToolbarDropdown/useDropdownControls.ts

```diff
@@ -16,9 +16,9 @@ const closeAllExceptSelectedOneListener = ({
   if (open) {
     const target = ev.target as HTMLElement;
     // TS2339: Property 'contains' does not exist on type 'ReferenceType'
-    // if (refs.reference.current?.contains(target)) {
-    //   return;
-    // }
+    if ((refs.reference.current as any)?.contains(target)) {
```

Comment:
Nit picking, maybe you can put this two conditions in one? they check the target on diff references only.
```suggestion
    const refsContainsTarget = (refs.reference.current as any)?.contains(target) || refs.floating.current?.contains(target);
    if (refsContainsTarget) {
        return ;
    }
```